### PR TITLE
Update verified publisher icon

### PR DESCRIPTION
**Pull request by parlough**
The icon was updated in https://github.com/dart-lang/pub-dev/pull/5734

Staged: https://dart-dev--pr4465-fix-new-verified-pub-1fvf9bmb.web.app/tools/pub/verified-publishers

### CODE REVIEW

File: src/tools/pub/verified-publishers.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ title: Verified publishers
 description: Learn what verified publishers are, and they're verified.
 ---
 
-The pub.dev verified publisher badge <img src="/assets/img/verified-publisher.svg" alt="pub.dev verified publisher logo"> lets you know that a package
+The pub.dev verified publisher badge <img src="/assets/img/verified-publisher.svg" width="20" height="20" alt="pub.dev verified publisher logo"> 
+lets you know that a package
 was published by a publisher whose identity has been verified.
```
Comment (atsansone) [inline src/tools/pub/verified-publishers.md:8]: **issue (non-blocking):** Prefer that this run-on sentence with passive voice be fixed at the same time.
```suggestion
The pub.dev verified publisher badge <img src="/assets/img/verified-publisher.svg" width="20" height="20" alt="pub.dev verified publisher logo"> 
lets you know that the Dart team verified the identity of the developer who published this package.
```

File: src/tools/pub/publishing.md [hunk 2]
```diff
@@ -90,7 +90,9 @@ Using a verified publisher has the following advantages:
 * The consumers of your package know that the publisher domain has been verified.
 * You can avoid having pub.dev display your personal email address.
   Instead, pub.dev displays the publisher domain and contact address.
-* A verified publisher badge <img src="/assets/img/verified-publisher.svg" alt="pub.dev verified publisher logo"> is displayed next to your package name on both search pages and individual package pages.
+* A verified publisher badge <img src="/assets/img/verified-publisher.svg" width="20" height="20" alt="pub.dev verified publisher logo"> 
+  is displayed next to your package name
+  on both search pages and individual package pages.
```
Comment (atsansone) [inline src/tools/pub/publishing.md:95]: **issue (non-blocking):** Update this copy to clean up this sentence.
```suggestion
* The pub site displays a verified publisher badge <img src="/assets/img/verified-publisher.svg" width="20" height="20" alt="pub.dev verified publisher logo"> 
  next to your package name on both search pages and individual package pages.
```

**Review by atsansone — APPROVED** [review]
LGTM % fixing two run-on sentences. No further review required.

File: src/tools/pub/verified-publishers.md (hunk 1 above)
Comment (parlough) [reply src/tools/pub/verified-publishers.md]: Sure. I changed Dart team to pub.dev as it is not manual verification but rather done automatically tied to DNS records.

**Pull request merged**